blog/content/post/2023-02-24-msi_ms7d25_v1.1.1_release.md

```diff
@@ -0,0 +1,207 @@
+  have been also extended with multiple suspend/resume cycles to avoid such
+  problem in the future.
+
+- [Intel XTU on Windows reports "The platform does not support overclocking" on the MSI PRO Z690-A WIFI DDR4 with a K-series Processor](https://github.com/Dasharo/dasharo-issues/issues/159)
+
+  The issue has been reported by community in the first days after v1.0.0
+  release. Surprisingly the Intel XTU utility reported that platform does not
+  support overclocking, despite all components had all the overclocking
+  capabilities. The culprit was the OC lock bit being set by default in FSP
+  which prevented Intel XTU from modifying any settings from Windows level, so
+  it reported the platform as unsupported. The OC lock changes the
+  initialization path inside FSP, so you may encounter some unexpected changes
+  in CPU configuration. Please do not hesitate to report them. One of such
+  unexpected changes was an override of CPU turbo limits and causing the CPU
+  frequency to be lower than maximum default value (and so the performance was
+  degraded).
+
+- [SATA ports malfunction or Hot-Plug function disabled](https://github.com/Dasharo/dasharo-issues/issues/315)
+
+  A feature request from community to enable SATA hotplug functionality.
+
+- Platform sometimes automatically powers on after power off
+
+  Noticed by one of Dasharo Team developers when dGPUs are plugged. It happened
+  that the dGPUs were sending ACPi SCI interrupts which are not currently
+  handled by ACPI code and it resulted in instant wake up of the board when
+  powering off. The SCI interrupts have been disabled on all PCI Express Root
+  Ports, until proper handling of SCI is implemented to fix the problem.
+
+- GPIO controller ACPI device yellow bang in Windows device manager
+
+  Despite previous attempts to fix the yellow bang of GPIO controller in
+  Windows Device Manager, the issue still persisted. The ACPI GPIO controller
+  device had all the definitions of GPIOs for Alder Lake-S, but apparently,
+  Windows was unable to properly initialize the GPIO because of the silent
+  dependency on the ACPI path of the device in the Windows GPIO driver. While
+  Linux had no problems with previous ACPI path, Windows could not properly
+  load drivers in correct sequence, thus the ACPI path had to be changed from
+  `\_SB.PCI0.GPIO` to `\_SB.GPIO` so that Windows will not depend on
+  enumeration and initialization of `\_SB.PCI0` ACPI device. Similar issues
+  were observed with TPM2 device ACPI path. When set wrong, the Windows 11
+  installer would not work. Now the yellow bang in Device Manager should be
+  gone for good.
+
+- Resource conflicts with chipset internal P2SB PCI device being incorrectly
+  defined and initialized in coreboot
+
+  Issue observed during the 2x dGPU memory allocation problems and GPIO
+  allocation problems. It occurred that coreboto did not reserve proper memory
```

Comment:
```suggestion
  allocation problems. It occurred that coreboot did not reserve proper memory
```

Reply:
https://github.com/3mdeb/news-and-ideas/pull/513/commits/0f29e7a980e1814bfdf6c3d8ab045c7ba6d63782#diff-b67e40bb2c6780be37b7e6cf6fc2a2a9f9ad977388795cc0c5bdfa6576dc52b6R170